code/code/cmd/cmd_egotrip.cc

```diff
@@ -88,11 +88,12 @@ std::map <spellNumT,ego_imm_blessing> init_ego_imm_blessing()
         AFFECT_DASH_BLESSING,
         APPLY_FOC,
         "<B>reflection<1>"); // Steal my blessing again, and I'll break you. -D
-  blessings[AFFECT_ANGUS_BLESSING]=
-    ego_imm_blessing("Angus",
-        AFFECT_ANGUS_BLESSING,
-        APPLY_WIS,
-        "<g>wisdom<1>");
+  blessings[AFFECT_SIDARTHA_BLESSING]=
+    ego_imm_blessing("Sidartha",
+        AFFECT_SIDARTHA_BLESSING,
+        APPLY_SPE,
+	APPLY_HASTE,	     
```

Comment:
Does this work?

Reply:
Works from my view unless Elmo has other concerns.

Reply:
What I mean is APPLY_HASTE isn't a thing defined in the code. APPLY_SPE is defined here: https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/enum.h#L333-L372

This code won't compile

Reply:
Added hitroll instead.